test/bats/tests/azure/azure_v1_secretproviderclass.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
 spec:
   provider: azure
   parameters:
+    usePodIdentity: "false"
+    useVMManagedIdentity: "false"
```

Comment:
Why are these fields required and set to `false` ? Aren't these defaults to false?